src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -933,4 +933,29 @@ public void func_146977_a( final Slot s )
 	{
 		this.drawSlot( s );
 	}
+
+	protected GuiScrollbar getScrollBar()
+	{
+		return this.myScrollBar;
+	}
+
+	protected void setScrollBar( final GuiScrollbar myScrollBar )
+	{
+		this.myScrollBar = myScrollBar;
+	}
+
+	protected List<InternalSlotME> getMeSlots()
+	{
+		return this.meSlots;
+	}
+
+	public static synchronized final boolean isSwitchingGuis()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)